### ISSUE TITLE
Question about providing Linux Drivers for Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by dfmaxwell** [Microsoft/msphpsql#156]
I'm wondering if in conjunction with the drivers that you've provided which align with the latest version of Ubuntu and PHP if you'd consider providing drivers specifically compiled against the current version of Ubuntu 16.04 LTS.

As of today 16.04.1 is the current LTS version.
PHP version is PHP 7.0.8-0ubuntu0.16.04.2
Apache version is Apache/2.4.18 (Ubuntu)

Thanks 

Update: The current version of the driver today (4.0.4) seems to be working fine with Ubuntu LTS 16.04.1 and this PHP version.

**Comment by meet-bhagdev** [comment]
@dfmaxwell our drivers should build and work against Ubuntu 16.04 and Ubuntu 16.04.1. Are you having trouble getting this to work?

**Comment by dfmaxwell** [comment]
I didn't try to build them because I'm using a package from the ubuntu 16.04.1 LTS distribution.  I had tried to use the nts .so package prior to 4.0.4 and it caused problems.  I saw you compile against PHP 7.0.10 and when I last tried it was the 4.0.3 release with PHP 7.0.9 and I presumed there was a minor incompatibility somewhere.  But 4.0.4 is working great and I like the changes I see.  I have a lot to convert and test from freetds/mssql over to this so will keep working and playing and observing how it goes and tracking the progress that is being made.
